homeworks/yehor.honcharenko_NOcode-developer/GamesProject/About/style.css

```diff
@@ -0,0 +1,181 @@
+body {
+  width: 100%;
+  margin: 0;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+}
+
+.wrapper {
+  display: grid;
+  grid-template-rows: auto 1fr auto;
+  min-height: 100vh;
+}
+
+/* Header */
+
+header {
+  grid-row: 1 / 2;
+  background: #EF4934;
+  padding: 8px 0;
+}
+
+.nav__container {
+  max-width: 1280px;
+  margin: 0 auto;
+  display: grid;
+  grid-template-columns: auto 1fr;
+  align-items: center;
+  box-sizing: border-box;
+}
+
+.logoTop {
+  width: 64px;
+  height: 64px;
+}
+
+ul {
+  display: grid;
+  grid-template-columns: repeat(2, auto);
+  grid-gap: 50px;
+  justify-content: end;
+}
+
+li {
+  list-style: none;
+}
+
+.navLink {
+  text-decoration: none;
+  color: #FFF;
+  font: 500 16/24px Roboto;
```

Comment:
This stuff doesn't work in web
<img width="1064" alt="image" src="https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/a56b4887-88e7-40d5-ab9b-08e6e7213a2b">

Reply:
fixed / done ✅